packages/graphql/lib/src/links/websocket_link/websocket_client.dart

```diff
@@ -166,7 +170,7 @@ class SocketClient {
 
```

Comment:
For graphql-transport-ws, connect needs to block before subscribe is sent, otherwise a 4401 error will be returned.

Replace `_connect` call in constructor with public `connect` call by invoker to allow blocking.